pkg/test/kube/kubectl.go

```diff
@@ -18,6 +18,8 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
+
+	"istio.io/istio/pkg/test/shell"
```

Comment:
Do we need this file for anything? Can we replace it with `accessor`?

Reply:
I am sure there is a way, but I am not aware of it at the moment. The only way I know to apply a Yaml file reliably is to use "kubectl apply". 

If we can figure out a way to do apply (just as kubectl does) without inheriting a lot of code, we should definitely do that.